src/test/java/com/fasterxml/jackson/core/io/BigDecimalParserTest.java

```diff
@@ -51,6 +52,18 @@ void longValidStringFastParse() {
         assertEquals(EXP, BigDecimalParser.parseWithFastParser(num.toCharArray(), 0, num.length()));
     }
 
+    @Test
```

Comment:
This is useful, although would probably be good to also test via `JsonParser` backends.